server/schedule/filter/filters_test.go

```diff
@@ -205,24 +251,33 @@ func (s *testFiltersSuite) TestIsolationFilter(c *C) {
 func (s *testFiltersSuite) TestPlacementGuard(c *C) {
 	opt := mockoption.NewScheduleOptions()
 	opt.LocationLabels = []string{"zone"}
-	tc := mockcluster.NewCluster(opt)
-	tc.AddLabelsStore(1, 1, map[string]string{"zone": "z1"})
-	tc.AddLabelsStore(2, 1, map[string]string{"zone": "z1"})
-	tc.AddLabelsStore(3, 1, map[string]string{"zone": "z2"})
-	tc.AddLabelsStore(4, 1, map[string]string{"zone": "z2"})
-	tc.AddLabelsStore(5, 1, map[string]string{"zone": "z3"})
+	testCluster := mockcluster.NewCluster(opt)
+	allStores := []struct {
+		storeID     uint64
+		regionCount int
+		labels      map[string]string
+	}{
+		{1, 1, map[string]string{"zone": "z1"}},
+		{2, 1, map[string]string{"zone": "z1"}},
+		{3, 1, map[string]string{"zone": "z2"}},
+		{4, 1, map[string]string{"zone": "z2"}},
+		{5, 1, map[string]string{"zone": "z3"}},
+	}
+	for _, store := range allStores {
+		testCluster.AddLabelsStore(store.storeID, store.regionCount, store.labels)
+	}
```

Comment:
IMO this looks harder to read compared to the original.

Reply:
Yeah...It actually make more noise to a code reader to figure out what we're doing here. I will change it to the old style.